lib/packwerk/load_paths.rb

```diff
@@ -5,7 +5,7 @@
 
 module Packwerk
   # Extracts the load paths from the analyzed application so that we can map constant names to paths.
-  module ApplicationLoadPaths
+  module LoadPaths
```

Comment:
Hm, now that I read this more closely, I think we should name this `RailsLoadPaths` and go the peer dependency route. We can make this interface generalized and switch the module out for something else later as needed (if we support other frameworks, for example).